src/services/userService.ts

```diff
@@ -11,6 +12,23 @@ import taggingUtils from '../lib/taggingUtils';
 import SetUpError, { NOT_SETUP } from '../lib/errors/SetupError';
 import { populateCommonKeyId } from '../lib/cryptoUtils';
 
+export interface User {
+  id: string;
+  commonKey: Object;
```

Comment:
Since the `commonKey` is a very common property of type `Object`, should we consider type definition for its properties or that is expected to be handled at some point by the `js-crypto`?

Reply:
Good point and yes to both. I added a basic type in here already but js-crypto should have a proper typing not only for its keys and that would be a bit more elaborate then.